frappe/desk/doctype/number_card/number_card.py

```diff
@@ -107,9 +110,13 @@ def get_percentage_difference(doc, filters, result):
 		return
 
 	previous_result = calculate_previous_result(doc, filters)
-	difference = (result - previous_result)/100.0
-
-	return difference
+	if previous_result == 0:
+		return NaN
```

Comment:
`NaN` won't work in python. We can return `None` here but we'll have to change some of the client side code too. Mind if I push those changes?

Reply:
Sure. If there is anything I need to do, do let me know!

In the meanwhile, I'll change NaN to None.

Reply:
Thanks! Pushed the changes. Also try to follow these conventions for your commit messages - https://github.com/pvdlg/conventional-commit-types

Reply:
Sure, will do, a newbie here. Thank you!